introduction.md

```diff
@@ -1 +1,6 @@
 Hi, I am Aswin Ramesh. My favourite Code School path is iOS.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?